test/mitmproxy/proxy/protocol/test_http2.py

```diff
@@ -508,8 +505,10 @@ def handle_server_event(cls, event, h2_conn, rfile, wfile):
         return True
 
     def test_body_size_limit(self):
-        self.config.options.body_size_limit = "20"
-        self.config.options._processed["body_size_limit"] = 20
+        self.options.body_size_limit = "20"
+
+        # FIXME: This should not be required?
+        self.options._processed["body_size_limit"] = 20
```

Comment:
No idea - if this code is from me, I probably just copied it from somewhere without knowing what it is doing...